src/trials.jl

```diff
@@ -290,18 +297,17 @@ function withtypename(f, io, t)
     end
 end
 
-function bindata(sorteddata, nbins, min, max)
-    Δ = (max - min) / nbins
-    bins = zeros(nbins)
-    lastpos = 0
-    for i ∈ 1:nbins
-        pos = searchsortedlast(sorteddata, min + i * Δ)
-        bins[i] = pos - lastpos
-        lastpos = pos
+function histogram_bindata(data, fences::AbstractRange)
```

Comment:
I re-wrote this, as it seemed the shortest path to making all the edge cases I invented work out (e.g. always highlight the correct median bar). It will be slower but no longer needs to sort the times, so I think in the end the whole thing comes out a similar speed.